Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Reflection.PortableExecutable;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson4
+    {
+        public static float globalHeight = 0f;
+        public static float globalWeight = 0f;
+        public static string bmiError = "Failed calculating BMI. Reason:";
+
+        public static string Information()
```

Comment:
Methods are actions and therefore should be named after verbs. Information is not informative enough. Please rename to PromptBmi or similar

Reply:
Thank you for the feedback. I renamed the methods to be more clear and concise as part of reworking the solution.